app/assets/javascripts/admin/models/admin-user.js.es6

```diff
@@ -168,6 +168,17 @@ const AdminUser = Discourse.User.extend({
     }).catch(popupAjaxError);
   },
 
+  disableSecondFactor() {
+    const self = this;
```

Comment:
We are migrating away from `self = this` as a pattern. Instead use ES6 arrow functions so that `this` points at the correct thing.